bin/node/runtime/src/lib.rs

```diff
@@ -887,6 +939,10 @@ parameter_types! {
 	pub const ImOnlineUnsignedPriority: TransactionPriority = TransactionPriority::max_value();
 	/// We prioritize im-online heartbeats over election solution submission.
 	pub const StakingUnsignedPriority: TransactionPriority = TransactionPriority::max_value() / 2;
+	pub const MaxAuthorities: u32 = 100;
+	pub const MaxKeys: u32 = 10_000;
+	pub const MaxPeerInHeartbeats: u32 = 10_000;
+	pub const MaxPeerDataEncodingSize: u32 = 1_000;
```

Comment:
[Refactoring](https://github.com/paritytech/substrate/pull/9654) of [pallet_im_online](https://docs.rs/pallet-im-online/latest/pallet_im_online/)